homeassistant/components/chacon_dio/manifest.json

```diff
@@ -5,6 +5,6 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/chacon_dio",
   "iot_class": "cloud_push",
-  "loggers": ["dio_chacon_api"],
+  "loggers": ["chacon_dio"],
```

Comment:
This should follow the logger in the library btw

Reply:
Done